C-Plus-Plus/other/Unque_no_3.cpp

```diff
@@ -0,0 +1,37 @@
+//UNIQUE NUMBER III

```

Comment:
@ishi0301 Update all single-line comments in one multiline comment. Something like this
```
/* 
...
*/
```